src/openrct2/localisation/Formatting.cpp

```diff
@@ -341,6 +341,11 @@ namespace OpenRCT2
                 buffer[i++] = static_cast<char>('0' + (num % 10));
                 num /= 10;
             }
+            // handle case where value has fewer sig figs than required decimal places
+            while (num == 0 && i < TDecimalPlace && i < sizeof(buffer))
+            {
+                buffer[i++] = static_cast<char>('0');
```

Comment:
Surely the static cast is not required.

Reply:
yes certainly not required, just put it there for consistency/readability with the prior line that makes use of '0' char literal as an offset: `static_cast<char>('0' + (num % 10))`. Removing the static cast probably won't have any effect on code generation, but if you'd prefer it be removed for style I can.

Reply:
I agree, just remove it if it isn't required.

Reply:
I have removed the explicit cast; I made my commit summary message slightly too long though and got caught by the CI linter; is there anything I need to do about that (e.g. commit --amend) or is it more of a reminder?

Reply:
Just amend the commit the Mac os ci has failed as well. Might be unrelated though.

Reply:
Mac CI mysteriously had a connection timeout during upload phase from what I could tell, might just be a random glitch with CI

Reply:
Yeah Mac CI always seems to have issues.